starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"> 
+<link rel="stylesheet" href="./css/main.css">
 <head>
   <title>Apple Pie Recipe</title>
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <header class="header">
+    <h1>Apple Pies</h1>
```

Comment:
Since your container already is a header, you don't really need to give it a class of "header". You can simply target the entire container like so:
```css
header {
    /* your styling here */
}
```